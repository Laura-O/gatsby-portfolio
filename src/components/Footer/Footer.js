import React, { Component } from 'react';
import Link from 'gatsby-link';
import classNames from 'classnames';
import styles from './styles.module.scss';
import github from '../../assets/icons/github.svg';
import linkedin from '../../assets/icons/linkedin.svg';
import twitter from '../../assets/icons/twitter.svg';

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <div className={styles.social}>
        <ul className={styles.list}>
          <li>
            <a href="https://github.com/Laura-O">
              <img className={styles.icon} src={github} alt="github" />
            </a>
          </li>
          <li>
            <a href="www.linkedin.com/in/lauraohrndorf">
              <img className={styles.icon} src={linkedin} alt="linkedin" />
            </a>
          </li>
          <li>
            <a href="https://twitter.com/laura_fyi">
              <img className={styles.icon} src={twitter} alt="twitter" />
            </a>
          </li>
        </ul>
      </div>
    </footer>
  );
};

export default Footer;
