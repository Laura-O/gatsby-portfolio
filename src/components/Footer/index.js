import React, { Component } from 'react';
import Link from 'gatsby-link';
import classNames from 'classnames';
import styles from './styles.module.scss';

const Footer = () => {
  return (
    <footer>
      <ul>
        <li>bla</li>
      </ul>
    </footer>
  );
};

export default Footer;
