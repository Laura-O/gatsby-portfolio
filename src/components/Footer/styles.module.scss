@import '../../scss/variables.scss';

footer {
  display: flex;
  justify-content: flex-end;
  padding: 10px 30px 15px 30px;
  background-color: $darkBackground;
  color: $colorWhite;

  li {
    display: inline-block;
    margin: 5px;
  }
}

.icon {
  width: 1.5em;
  height: 1.5em;
}
.social {
  align-content: flex-end;
}
