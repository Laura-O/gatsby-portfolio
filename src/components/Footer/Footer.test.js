import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

test('rendering', () => {
  const wrapper = shallow(<Footer path="/something" />);
  expect(wrapper).toMatchSnapshot();
});
