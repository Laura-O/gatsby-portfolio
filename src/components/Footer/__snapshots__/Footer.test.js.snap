// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<footer>
  <div>
    <ul>
      <li>
        <a
          href="https://github.com/Laura-O"
        >
          <img
            alt="github"
            src="test-file-stub"
          />
        </a>
      </li>
      <li>
        <a
          href="www.linkedin.com/in/lauraohrndorf"
        >
          <img
            alt="linkedin"
            src="test-file-stub"
          />
        </a>
      </li>
      <li>
        <a
          href="https://twitter.com/laura_fyi"
        >
          <img
            alt="twitter"
            src="test-file-stub"
          />
        </a>
      </li>
    </ul>
  </div>
  <div>
    <small>
      <a
        href="/legal"
      >
        Legal notice
      </a>
    </small>
  </div>
</footer>
`;
